specs/fulu/validator.md

```diff
@@ -43,6 +49,56 @@ document and used throughout.
 | - | - | - |
 | `VALIDATOR_CUSTODY_REQUIREMENT` | `8` | Minimum number of custody groups an honest node with validators attached custodies and serves samples from |
 | `BALANCE_PER_ADDITIONAL_CUSTODY_GROUP` | `Gwei(32 * 10**9)` | Balance increment corresponding to one additional group to custody |
+| `MAX_CELLS_PER_EXT_BLOB` | `1024` | Corresponds to a cell with 8 field elements per blob |
+
+## Helpers
+
+### `BlobsBundle`
+
+*[Modified in Fulu:EIP7594]*
+
+The `BlobsBundle` object is modified to include cell KZG proofs.
+
+```python
+@dataclass
+class BlobsBundle(object):
+    commitments: List[KZGCommitment, MAX_BLOB_COMMITMENTS_PER_BLOCK]
+    proofs: List[KZGProof, MAX_BLOB_COMMITMENTS_PER_BLOCK]
```

Comment:
prefer a new version with proof's semantics changed or clarity into should we expect empty proofs

basically this should be ssz container friendly to figure out what container type is if we have to figure out the type of container by inspection. If we infer the types some other way (by the slot involved for e.g.) we should just override proofs then with the new list length

Reply:
Yes, I also prefer a new `fulu.BlobsBundle` type without `proofs`. Right now, it is like this to showcase Francis' proposed solution. I'll go with whatever client teams decide.